docs/product/dev-toolbar/setup.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: "Set Up"
+sidebar_order: 10
+description: "Get started with Sentry's Dev Toolbar, bringing critical Sentry insights and tools into your web app to help your team troubleshoot more effectively.."
+---
+
+<Alert level="info">
+  The Dev Toolbar is currently in **beta**. Beta features are still in progress and may have bugs. Please reach out on
+  [GitHub](https://github.com/getsentry/sentry-toolbar/issues) if you have any feedback or concerns.
+</Alert>
+
+## Set Up Your Web App
+
+<Alert title="Note" level="warning">
+    [Enabling tracing](/platforms/javascript/tracing/) is a prerequisite for having page-aware panels, as it allows us to collect page-specific issues and feedback through transactions.
+</Alert>
+
+You need to complete two steps to get the toolbar rendered on the page:
+1. Add or dynamically inject `<script defer src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>` into your web app. This script uses the `defer` attribute so that it does not block document parsing. The script tag can be inserted at the bottom of the page so it doesn’t block other critical Javascript.
+2. Call `window.SentryToolbar.init(initConfig)` to set up a toolbar instance on each page where you want to see the Dev Toolbar.
```

Comment:
This makes it feel like you have to add both to every page instead of just the `window.SentryToolbar.init(initConfig)`. Is that true? If not, we should consider removing the first script from this code snippet.

Reply:
@ryan953 I'm not sure how to rewrite so that it's clear that only the init needs to be added to every page. Any ideas?

Reply:
Both need to exist together whenever, and can be inserted whenever a full pageload happens. So if we're talking nextjs they can go in the custom [`_document.tsx`](https://nextjs.org/docs/pages/building-your-application/routing/custom-document) file, or if it's laravel it'll go into [`master.blade.php`](https://laravel.com/docs/5.0/templates) for example.


Only having `<script src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>` will load up the JS bundle, but won't render anything. While calling `window.SentryToolbar.init()` without the other script tag existing will cause an error.  So they both have to exist, in this order on the page.